lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2542,6 +2542,102 @@ class ConvertAtenOnesOp : public OpConversionPattern<AtenOnesOp> {
 };
 } // namespace
 
+namespace {
+
+static Type getTypeFromDTypeInteger(MLIRContext *context, int64_t dtypeInt) {
```

Comment:
actually, we have this helper in RefineTypes. Could you help to put it to `https://github.com/llvm/torch-mlir/blob/main/lib/Dialect/Torch/Utils/Utils.cpp` so that this helper can be shared.

Reply:
Refactored the code such that this function is no longer being used in TorchToLinalg.cpp